### ISSUE TITLE
Custom pop up menu

### ISSUE THREAD

**Issue by 312654929** [JideGuru/epub_viewer#56]
Issue / Feature -

How to implement custom pop-up menu, such as highlight, note, and can I add menu options myself?
Is the WebView used to show the content of chapter

Platform (android, iOS or both) - both
Crash / Error - 
Steps to reproduce / Describe in detail -

**Comment by AliReza-Pro-grammer** [comment]
this is based on folio and as far as i have understood,
you need to make changes to that library.
if anyone know i am wrong, please say so. i really want to customize this one myself.

**Comment by 312654929** [comment]
> this is based on folio and as far as i have understood,
> you need to make changes to that library.
> if anyone know i am wrong, please say so. i really want to customize this one myself.

Hi,Now I don't need this project for the time being. I use readium mobile toolkit for secondary development

**Comment by syedecryptr** [comment]
> this is based on folio and as far as i have understood,
> you need to make changes to that library.
> if anyone know i am wrong, please say so. i really want to customize this one myself.

@AliReza-Pro-grammer 

Yes, this plugin is using a custom Folioreader [library](https://github.com/JideGuru/Folioreader-Android) 
you have to change in that library for any custom feature.
For example, I recently added a bookmark feature in the reader which you can refer to [here]https://github.com/JideGuru/Folioreader-Android/pull/1).

**Comment by carman247** [comment]
Can someone help me disable text selection? e.g. prevent the pop-up menu that has COPY / PAST / DEFINE / SHARE

**Comment by Strateg766** [comment]
> > this is based on folio and as far as i have understood,
> > you need to make changes to that library.
> > if anyone know i am wrong, please say so. i really want to customize this one myself.
> 
> @AliReza-Pro-grammer
> 
> Yes, this plugin is using a custom Folioreader [library](https://github.com/JideGuru/Folioreader-Android)
> you have to change in that library for any custom feature.
> For example, I recently added a bookmark feature in the reader which you can refer to [here](https://github.com/JideGuru/Folioreader-Android/pull/1).

please give a repo of your epub viewer with the addition of bookmarks

**Comment by Strateg766** [comment]
> > this is based on folio and as far as i have understood,
> > you need to make changes to that library.
> > if anyone know i am wrong, please say so. i really want to customize this one myself.
> 
> @AliReza-Pro-grammer
> 
> Yes, this plugin is using a custom Folioreader [library](https://github.com/JideGuru/Folioreader-Android)
> you have to change in that library for any custom feature.
> For example, I recently added a bookmark feature in the reader which you can refer to [here](https://github.com/JideGuru/Folioreader-Android/pull/1).

please give a repo of your epub viewer with the addition of bookmarks

**Comment by syedecryptr** [comment]
> > > this is based on folio and as far as i have understood,
> > > you need to make changes to that library.
> > > if anyone know i am wrong, please say so. i really want to customize this one myself.
> > 
> > 
> > @AliReza-Pro-grammer
> > Yes, this plugin is using a custom Folioreader [library](https://github.com/JideGuru/Folioreader-Android)
> > you have to change in that library for any custom feature.
> > For example, I recently added a bookmark feature in the reader which you can refer to [here](https://github.com/JideGuru/Folioreader-Android/pull/1).
> 
> please give a repo of your epub viewer with the addition of bookmarks

[Here](https://github.com/syedecryptr/epub_viewer)
